src/components/top_bar/top_bar.xml

```diff
@@ -116,7 +116,11 @@
               </div>
             </div>
             <div class="o-divider"/>
-            <FontSizeEditor onToggle="() => this.onClick()" dropdownStyle="this.dropdownStyle"/>
+            <FontSizeEditor
+              class="'o-tool'"
+              onToggle="() => this.onClick()"
```

Comment:
```suggestion
              onToggle.bind="this.onClick"
```
Otherwise, the child (`FontSizeEditor`) is always re-rendered every time this component is rendered because the reference of `onToggle` is always different (the anonymous function is recreated at every render)